main/EDInitMod.F90

```diff
@@ -760,7 +790,11 @@ subroutine init_cohorts( site_in, patch_in, bc_in)
     real(r8) :: m_sapw     ! Generic mass for sapwood [kg]
     real(r8) :: m_store    ! Generic mass for storage [kg]
     real(r8) :: m_repro    ! Generic mass for reproductive tissues [kg]
-    real(r8) :: stem_drop_fraction
+    real(r8) :: elongf_leaf       ! Leaf elongation factor
+    real(r8) :: elongf_fnrt       ! Fine-root "elongation factor"
+    real(r8) :: elongf_stem       ! Stem "elongation factor"
```

Comment:
I actually replaced the local variables with the cohort structure. In any case, the code uses elongation factors for leaves, fine roots and stems for all PFTs (woody or non-woody). However, in practice only non-woody species have time-varying elongation factors for stems, because stem_drop_fraction cannot be different from zero for woody plants (the codes issues an error).